src/control/controlobject.cpp

```diff
@@ -14,7 +14,7 @@ ControlObject::ControlObject(const ConfigKey& key,
         bool bTrack,
         bool bPersist,
         double defaultValue)
-        : m_key(key) {
+        : m_key(std::move(key)) {
```

Comment:
We cannot move form a const &. I guess this std::move which boils down to a cast is ignored. 
In this case, we need to pass the key by value to make use of the full benefit of avoiding reference counting.